### PR TITLE
Stemmer clean code

### PR DESCRIPTION
**Pull request by BozorgmehrZia**

### CODE REVIEW

**Comment by mohammadKhalafi** [comment]
salam
in changeSuffixFinishSecondStep method "return true" is duplicated code, and you can use default for your switch case to return false and return true at the end of method.

**Comment by ARB1380** [comment]
salam
in function checkDoubleConsonantSequence the second thing in the if part( char[index] != char[index - 1] can be put in another method and just call in this if

**Comment by BozorgmehrZia** [comment]
> salam in function checkDoubleConsonantSequence the second thing in the if part( char[index] != char[index - 1] can be put in another method and just call in this if
ino bishtar tozih midi

**Comment by ARB1380** [comment]
salam
in function constantVowelConstantSequence the return part can be put in another function

**Comment by ARB1380** [comment]
> > salam in function checkDoubleConsonantSequence the second thing in the if part( char[index] != char[index - 1] can be put in another method and just call in this if
> > ino bishtar tozih midi

ye tabe ye khati masalan mese hamoon isContant ke boolean bar migardoone bashe in made nazaram bood

**Comment by mohammadKhalafi** [comment]
in method checkConsonantForI the logic created by "if,else" is !(a ^ b)

**Comment by mohammadKhalafi** [comment]
تست فارسی

File: src/Stemmer.java [hunk 1]
```diff
@@ -196,137 +174,213 @@ private void setto(String s)
...
-            if (ends("ou")) break; return;
-        /* takes care of -ous */
-        case 's': if (ends("ism")) break; return;
-        case 't': if (ends("ate")) break;
-            if (ends("iti")) break; return;
-        case 'u': if (ends("ous")) break; return;
-        case 'v': if (ends("ive")) break; return;
-        case 'z': if (ends("ize")) break; return;
-        default: return;
+    private boolean changeSuffixFinishFirstStep(char ch){
```
Comment (MKasaei00) [inline src/Stemmer.java:283]: <div dir='rtl' >
کامنت‌های توابع رو به فرمت مشخص javadoc بنویسید
</div>

**Review by MKasaei00 — CHANGES_REQUESTED** [review]

**Review by MKasaei00 — APPROVED** [review]

**Pull request merged**